ui/src/app/pages/access/data-access-requirements.spec.tsx

```diff
@@ -235,12 +235,7 @@ describe('DataAccessRequirements', () => {
     const newModules = fp.map(
       (moduleStatus) => ({
         ...moduleStatus,
-        bypassEpochMillis:
-          // profile and publication are not bypassable.
-          moduleStatus.moduleName === AccessModule.PROFILECONFIRMATION ||
-          moduleStatus.moduleName === AccessModule.PUBLICATIONCONFIRMATION
-            ? null
-            : bypassFn(),
```

Comment:
Like the above, I missed this test file in the original PR but could have updated it there.